relay_handler.js

```diff
@@ -358,6 +376,8 @@ function handleFrameLazily(frame) {
     // - v2.Types.CallRequestCont
     var self = this;
 
+    var now = self.channel.timers.now();
```

Comment:
I'm dubious of this `now` pattern.

I feel like `self.start` / `self.end` etc is better pattern.